java/clients/src/main/java/sleeper/status/report/compaction/job/StandardCompactionJobStatusReporter.java

```diff
@@ -56,6 +57,7 @@ public StandardCompactionJobStatusReporter(PrintStream out) {
         stateField = tableFactoryBuilder.addField("STATE");
         createTimeField = tableFactoryBuilder.addField("CREATE_TIME");
         jobIdField = tableFactoryBuilder.addField("JOB_ID");
+        inputFilesCount = tableFactoryBuilder.addNumericField("TOTAL_FILES");
```

Comment:
Total files seems to suggest it might include the output files as well. We could just say input files.